kymatio/scattering1d/scattering1d.py

```diff
@@ -619,7 +619,7 @@ def scattering(x, psi1, psi2, phi, J, pad_left=0, pad_right=0,
         Whether to compute the 2nd order or not. Defaults to False.
     average_U1 : boolean, optional
         whether to average the first order vector. Defaults to True
-    size_scat : dictionary or int, optional
+    size_scattering : dictionary or int, optional
```

Comment:
it would be good to take this opportunity to document how to use it as a `dict` input

Reply:
Not in this PR, I think. At any rate, this function is an internal one (not to be used publicly), so its documentation is not top priority. We may even integrate it into the `Scattering1D` class.